### ISSUE TITLE
MAUI views in Native projects

### ISSUE THREAD

**Issue by thefex** [dotnet/maui#7950]
### Description

Is there a built-in support to embeeding MAUI views inside Native projects as in Xamarin Forms (https://docs.microsoft.com/en-us/xamarin/xamarin-forms/platform/native-forms) ?

I'd love to mix MAUI with my native apps as I do with Forms. This will increase amount of developers investing in MAUI related controls and in some cases make gradual migration to the MAUI possible.

### Public API Changes

```csharp
// initialization
Maui.Initialize(this) // Application / AppDelegate class

var mauiView = new MauiPage();

// iOS
mauiView.CreateViewController(); 

// Android
mauiView.CreateFragment(this);
```

### Intended Use-Case

- Embeeding MAUI in Native Xamarin applications.
- Migration in legacy projects (new views written within MAUI)

**Comment by drasticactions** [comment]
Yeah, that's already supported. https://github.com/dotnet/maui/pull/5218

The larger issue is that I don't believe it's documented beyond what's in this PR. That needs to be written. Otherwise, it's there and it should work.

**Comment by jfversluis** [comment]
Doesn't seem there is much for us to do here. @davidbritch ping for the potential docs on this to make sure it's on the list.
